README.md

```diff
@@ -1,29 +1,36 @@
 # improved-initiative
 
-*Combat tracker for Dungeons and Dragons (D&amp;D) 5th Edition*
+_Combat tracker for Dungeons and Dragons (D&amp;D) 5th Edition_
 
 The official Improved Initiative app lives at https://www.improved-initiative.com
 
 ## Local Development
 
 ### Requirements
 
-- node.js 12+
-- npm
+- [Node.js](https://nodejs.org/en/) (v12+)
+- [Visual Studio Code](https://code.visualstudio.com/) (VS Code)
```

Comment:
Visual Studio Code is not a requirement. It's listed in CONTRIBUTING.md as a workflow suggestion, though.

Reply:
I looked at your notes in the commit for this; I agree that it's helpful to include the VS Code instructions, but let's call it a "recommendation" rather than a requirement.

Reply:
For me I couldn't get the development environment working in any way other than through doing it with the VS Code debugger. That is why I listed it as a requirement rather than a suggestion.

Reply:
I think a lot of my problems were caused by me trying to use Git Bash for the console commands instead of Powershell. When I switched to Powershell I started having an easier time getting things working.

I need to make updates to the readme.

Reply:
Updated the readme to not be so heavily targeted at VS Code.